components/nifosg/controller.hpp

```diff
@@ -48,13 +48,14 @@ namespace NifOsg
     {
     public:
         typedef typename MapT::ValueType ValueT;
+        typedef typename MapT::MapType InnerMapType;
 
         ValueInterpolator()
             : mDefaultVal(ValueT())
         {
         }
 
-        ValueInterpolator(boost::shared_ptr<const MapT> keys, ValueT defaultVal = ValueT())
```

Comment:
There's an architectural problem with the serializer requiring me to remove const here. :-(   There might be a way to work around it, but it's not immediately obvious to me.